app/views/NewHomeView.coffee

```diff
@@ -110,6 +110,8 @@ module.exports = class NewHomeView extends RootView
     $(window).on 'resize', @fitToPage
     @fitToPage()
     setTimeout(@fitToPage, 0)
+    if document.location.hash is '#create-account'
+      @$('#create-account-link').click()
```

Comment:
For this kind of thing I'd probably feel better about calling a function that the click handler for that also calls, to have a bit better separation of concerns. Doesn't need fixing now.

`
if document.location.hash is '#create-account'
  @openCreateAccountModal()
`

Especially since the handler for that actually listens to a different tag, so you need to hunt a little to find it: https://github.com/codecombat/codecombat/blob/create-account-redesign/app/views/core/RootView.coffee#L28